_overviews/contribute/add-guides.md

```diff
@@ -79,24 +101,93 @@ indented example:
     case class Foo(x: Int)
 ~~~
 
-Tables of contents will be automatically generated in a sidebar for your document, and syntax highlighting
-is provided.
+### Scala 2 vs Scala 3
 
-### Criteria for Docs to be Accepted
+Sometimes you would like to compare between Scala 2 and Scala 3 in a document, for example in
+our [Hello World][hello-world] chapter of the Scala Book. Here is an example of how you
+can generate the same tabs in markdown with the `tabs` directive and class `tabs-scala-version`:
 
-The goal of this documentation repository is to be highly curated, rather than the approach by other community-driven
-documentation platforms, like wikis. Therefore, to be included on [docs.scala-lang.org][home], a document must:
+<!-- {% raw  %} -->
+~~~liquid
+{% tabs hello-world-demo class=tabs-scala-version %}
 
-- **"fit in"** to the repository (_i.e.,_ it should not be a complete duplicate of another article),
-- **be polished**, i.e. it must be thorough, complete, correct, and organized; written as an article to be understood
-  by many users.
-- **be maintained**, if the document might require revisions from time to time, be prepared to keep it up to date, or
-nominate someone to take ownership.
+{% tab 'Scala 2' for=hello-world-demo %}
+```scala
+object hello extends App {
+  println("Hello, World!")
+}
+```
+{% endtab %}
 
-If you have something you're thinking about contributing, or that you're thinking about writing in order to contribute
--- we'd love to consider it! Please don't hesitate to use GitHub issues and pull requests and the
-`#scala-contributors` room [on Discord](https://discord.com/invite/scala) for any questions, concerns,
-clarifications, etc.
+{% tab 'Scala 3' for=hello-world-demo %}
+```scala
+@main def hello() = println("Hello, World!")
+```
+{% endtab %}
+
+{% endtabs %}
+~~~
+<!-- {% endraw  %} -->
+
+It is crucial that you use the `tabs-scala-version` class to benefit from some cool user interactions:
```

Comment:
I would like to try - the only issue is that the inner tabs seem to be evaluated before the outer tab section, (and evaluation of inner tabs is done by the framework, rather than directly from the outer tabs) also the data for the inner tabs is added to a huge global hash table so we do need the id to be passed to the inner tabs somehow to avoid collisions

Reply:
OK. That sounds complicated then, let’s keep things as they are…